R/apply_frmt_methods.R

```diff
@@ -212,7 +223,10 @@ apply_frmt.frmt_combine <- function(frmt_def, .data, values, mock = FALSE, param
     ) %>%
     mutate(
       .is_all_missing =  all_missing(fmt_param_vals,.)
-    )
+    ) %>%
+    ## after .is_all_missing so that can be tabulated first
+    replace_na(missing_param_replacements)
```

Comment:
This needs an if around it. Cause at the moment it will cover up NA when missing isn't set

Reply:
There is also an issue when a frmt_when is used. Do we need to add a missing option to frmt_when as well?

Reply:
That is simple enough to sort out. for some reason I thought we converted NA's into "". WRT frmt_when including a missing case, that seems reasonable. Alternatively for the "default" case, it could be expected to drop the missing value in there.